bcgov/Unity
# Dev To Main Release 2024-02-12

QA Testable

[AB#22911](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/22911) - (Bugfix) Community Population Field Numeric Handling
[AB#15433](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/15433) - (Bugfix) Update tag creation behavior on Save to create incomplete tag
[AB#22943](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/22943) - (Bugfix) Flip the initials in the User Badge generated from Display Name
[AB#22890](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/22890) - (Bugfix) Mapping fixes - add support for tables
[AB#22931](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/22931) - (Bugfix) Complete Sub-Status missing from LOV
[AB#22985](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/22985) - (Bugfix) Sub-Status needs to map to Recommendation
[AB#22953](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/22953) - (Bugfix) Manage Columns list Recommendation should be renamed to Sub-Status
[AB#22942](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/22942) - (Bugfix) Correct mismatch in dates captured vs displayed in Applications List
[AB#22960](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/22960) - (Bugfix) Fix Total Project Budget % calculation errors
[AB#23018](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23018) - (Bugfix) Droplists/Display Orgbook off applicant
[AB#23012](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23012) - Update electoral Districts Seed script to 2024 version
[AB#9228](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/9228) - Show owner and assignee in details panel
[AB#22933](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/22933) - Remove Business Area Admin Role as a seeded default role
[AB#9832](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/9832) - Application Tags Donut Chart in Dashboard
[AB#13296](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/13296) - Manage column list updated columns

Non QA

[AB#10628](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/10628) - User assignment feature cleanup (remove unused code)
